<launch>
	
	<!-- Launch UR node -->
	<node pkg="ur" type="ur_node.py" name="ur_2">
		
		<!-- robot parameters -->
		<param name="robot_ip" value="192.168.2.8"/>
		<param name="robot_port" value="30001"/>
		<param name="robot_data_port" value="30003"/>
		
		<!-- topic refresh ratio (per second) -->
		<param name="rate" value="10"/>
		
	</node>
	
</launch>
